packages/rancher-sriov/generated-changes/overlay/app-README.md

```diff
@@ -0,0 +1,14 @@
+# Rancher SR-IOV Network Operator
+
+This chart is based on the upstream [k8snetworkplumbingwg/sriov-network-operator](https://github.com/k8snetworkplumbingwg/sriov-network-operator) project. The chart deploys the SR-IOV Operator and its CRDs. It allows users to deploy SR-IOV CNI, which can be used, together with [Multus](https://github.com/k8snetworkplumbingwg/multus-cni), to provide high performing extra network interfaces to pods.
+
+
+The chart installs the following components:
+
+ - SR-IOV Operator - The operator helps provision and configure SR-IOV CNI plugin and Device plugin. It requires Multus to operate correctly.
+ - SR-IOV Network Config Daemon - Deployed by the operator, this daemon is responsible for discovering the SR-IOV NICs on each node
```

Comment:
nit: even if you don't take these changes, make sure you either add punctuation to the second bullet point or remove it from the first.

```suggestion
 - SR-IOV Operator - An operator that helps provision and configure CNI or Device plugins (requires Multus)
 - SR-IOV Network Config Daemon - A Daemon deployed by the Operator that discovers SR-IOV NICs on each node
```

Reply:
I partially applied your suggestion. I think we should have `provision and configure SR-IOV CNI plugin and SR-IOV Device plugin`. If we remove SR-IOV, the user could think that any CNI or any Device Plugin can be deployed.

What do you mean by adding punctuation to the second bullet?

Reply:
You resolved the issue. I mean that the first had a `.` at the end and the second did not 😄